test/cpp/microbenchmarks/BUILD

```diff
@@ -331,60 +270,42 @@ grpc_cc_library(
     deps = [":helpers_secure"],
 )
 
-grpc_cc_test(
+grpc_cc_benchmark(
     name = "bm_fullstack_unary_ping_pong",
     size = "large",
     srcs = [
         "bm_fullstack_unary_ping_pong.cc",
     ],
-    args = grpc_benchmark_args(),
-    tags = [
-        "no_mac",  # to emulate "excluded_poll_engines: poll"
-        "no_windows",
-    ],
     deps = [":fullstack_unary_ping_pong_h"],
 )
 
-grpc_cc_test(
+grpc_cc_benchmark(
     name = "bm_fullstack_unary_ping_pong_chaotic_good",
-    size = "large",
     srcs = [
         "bm_fullstack_unary_ping_pong_chaotic_good.cc",
     ],
-    args = grpc_benchmark_args(),
-    tags = [
-        "no_mac",  # to emulate "excluded_poll_engines: poll"
-        "no_windows",
-    ],
     deps = [
         ":fullstack_unary_ping_pong_h",
         "//:grpcpp_chaotic_good",
     ],
 )
 
-grpc_cc_test(
+grpc_cc_benchmark(
     name = "bm_chttp2_hpack",
     srcs = ["bm_chttp2_hpack.cc"],
-    args = grpc_benchmark_args(),
     external_deps = [
         "absl/log:check",
     ],
-    tags = [
-        "no_mac",
-        "no_windows",
-    ],
     uses_event_engine = False,
-    uses_polling = False,
     deps = [
         ":helpers",
         "//src/core:slice",
     ],
 )
 
-grpc_cc_test(
+grpc_cc_benchmark(
     name = "bm_opencensus_plugin",
     srcs = ["bm_opencensus_plugin.cc"],
-    args = grpc_benchmark_args(),
     language = "C++",
```

Comment:
For consistency, you probably want to remote the language here. Why did you remove it from the other places, though?

Reply:
it's a no-op for these things